### PR TITLE
Fix broken installation

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Fix broken installation since module was renamed
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | PrestaShop SA
| How to test?      | try installing the module, it should work

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @matthieu-rolland!

This is your first pull request on the PrestaShop project. Thank you, and welcome to this Open Source community!

**Review by boherm — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
DO NOT MERGE right now, @boherm will do some tests for the kanban bot on it, I'll merge it when it can be merged, thanks

**Comment by elyesnaouar** [comment]
Test OK

**Pull request merged**
